gtp_cmd_handlers.py

```diff
@@ -0,0 +1,340 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime, timedelta
+import sys
+import itertools
+import json
+import sgf_wrapper
+import go
+import coords
+
+
+def dbg(fmt, *args):
+    print(fmt % args, file=sys.stderr, flush=True)
+
+
+def translate_gtp_color(gtp_color):
+    if gtp_color.lower() in ["b", "black"]:
+        return go.BLACK
+    if gtp_color.lower() in ["w", "white"]:
+        return go.WHITE
+    raise ValueError("invalid color {}".format(gtp_color))
+
+
+class BasicCmdHandler(object):
+    """GTP command handler for basic play commands."""
+
+    def __init__(self, player, courtesy_pass=False):
+        self._komi = 6.5
```

Comment:
probably want to default to 7.5

Reply:
Agreed, but the 6.5 comes from the original code so I'm going to leave it unchanged in this PR, since in theory it shouldn't change any behaviour.